README.md

```diff
@@ -8,7 +8,7 @@ It is intended that `wagtail-generic-chooser` will be expanded to cover all the
 
 ## Requirements
 
-Wagtail 2.4 or higher
+Wagtail 2.15 or higher
```

Comment:
I don't think we need to change this, as we're not breaking anything against older versions (even if they're out of support).